build-source-tarball.sh

```diff
@@ -87,6 +87,9 @@ git submodule foreach --quiet --recursive '
         git --work-tree="$TARBALL_SUBMODULE_PATH" checkout -- .
     fi'
 
+echo 'Removing binaries from tarball src...'
+find $TARBALL_ROOT/src \( -type f \( -iname *.dll -o -iname *.exe \) \) -exec rm {} \;
```

Comment:
I think it would be a good idea to log the removed files to a txt file in the tarball, for diagnosability if it goes wrong--especially if `file -bi` works out and we use that.

Reply:
Oh, interesting, I assumed `file` was part of some basic toolset. Yeah, still ok with merging this as-is.

And actually, since this only happens in `src`, it's all tracked by git, so it should be easy to whip up a list of removed files after the fact if needed no matter what.